data/campaigns/emp04.wmf/scripting/texts.lua

```diff
@@ -57,6 +57,11 @@ obj_clear_roads = {
    ),
 }
 
+obj_clear_roads_hint = rt(objective_text(_"Clear Road Network",
+   li(_[[Resolve the chaotic road network by clearing all unnecessary roads.]]) ..
+   li_arrow(_[[Ensure there are not more than 3 dead ends or flags with more than 4 roads.]])
```

Comment:
At first I interpreted this as "not more than 3 (dead ends or flags with more than 4 roads)", and so I thought it was different from amalea_20 "and not more than one flag with more than four roads". It would be better if it used the same wording.

Reply:
By the way, I still think it would be nice to clarify here (it's not necessary in Amalea's text) that building entrances count as roads.

Reply:
I tried to shorten the message in the objectives. During translation to german I already realized this could be misunderstood. However the player already received Amaleas reminder when this is displayed so he should know if he read the message. we might change it to ... dead ends or 1 flag with .... though. regarding the explanation of the house entrances, I believe if at alol this should go to Amaleas text and not the objective.